calculator/main.py

```diff
@@ -3,20 +3,22 @@
 from calculator.operations import *
 from calculator.exceptions import *
 
+# xyz = create_new
 
-def create_new_calculator(operations=None):
+def create_new_calculator(operations={}):
```

Comment:
This is a bad idea. Subtle (and advanced) bug, but here is for you guys to read about it (Mistake 1 in this list): https://www.toptal.com/python/top-10-mistakes-that-python-programmers-make

Please remind me on Saturday to talk about this.